src/Summoner/Tree.hs

```diff
@@ -35,7 +38,10 @@ showOne leader tie arm (Dir fp (sortWith treeFp -> trees)) =
     nodeRep : showChildren trees (leader <> extension)
   where
     nodeRep :: Text
-    nodeRep   = leader <> arm <> tie <> toText fp
+    nodeRep
+        = leader <> arm <> tie <> toText
+            (setSGRCode [SetConsoleIntensity BoldIntensity]
+                <> fp <> "/" <> setSGRCode [Reset])
```

Comment:
Could you please move it into separate function called `bold` so it looked like:

```haskell
nodeRep = leader <> arm <> tie <> bold (fp <> "/")
```